src/app/app.module.ts

```diff
@@ -44,6 +46,9 @@ export const ROUTES: Routes = [{
     canActivate: [AuthService],
     canActivateChild: [AuthService],
 }, {
+    path: 'error',
+    component: ErrorComponent,
```

Comment:
Errors should be displayed on the same path they happened. So you can refresh and see if it goes away